src/pages/[[...slug]].tsx

```diff
@@ -64,12 +64,9 @@ export async function getStaticPaths(
     }
   }
 
-  const storyListingPaths = await getAllStoryListingStaticPaths(context)
-  const storyPaths = await getStaticPathsByResourceType(
-    RESOURCE_TYPES.STORY,
-    context
-  )
-  const qaPaths = await getStaticPathsByResourceType(RESOURCE_TYPES.QA, context)
+  const storyListingPaths = await getAllStoryListingStaticPaths()
```

Comment:
not a blocker to this PR at all, but i think eventually we'll want to genericize this so we can use it for other Listing page types too (Events Listing, Publication Listing, Press Release listing, etc) instead of unique functions for each.